app/services/GetMessageStatisticsService.scala

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import config.AppConfig
+import connectors.emcsTfe.GetMessageStatisticsConnector
+import featureswitch.core.config.FeatureSwitching
+import models.requests.UserRequest
+import models.response.MessageStatisticsException
+import models.response.emcsTfe.GetMessageStatisticsResponse
+import uk.gov.hmrc.http.HeaderCarrier
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class GetMessageStatisticsService @Inject()(connector: GetMessageStatisticsConnector,
+                                            override val config: AppConfig
+                                           )(implicit ec: ExecutionContext) extends FeatureSwitching {
+
+  def getMessageStatistics(ern: String)(implicit hc: HeaderCarrier, request: UserRequest[_]): Future[Option[GetMessageStatisticsResponse]] = {
```

Comment:
```suggestion
  def getMessageStatistics(ern: String)(implicit hc: HeaderCarrier): Future[Option[GetMessageStatisticsResponse]] = {
```
`request` is never used so can be removed

Reply:
Addressed at https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/350/commits/fa149054706fb8b20ca731b0d832b7e76fb43759